sqlalchemy_utils/view.py

```diff
@@ -60,7 +65,7 @@ def create_table_from_selectable(
         )
         for c in get_columns(selectable)
     ] + indexes
-    table = sa.Table(name, metadata, *args, **kwargs)
+    table = sa.Table(name, metadata, schema=schema, *args, **kwargs)
```

Comment:
Python will crash with a `SyntaxError` when it encounters this line because all keyword arguments (`schema=schema`) must appear positional arguments (`*args` is the last positional argument in this line).

Reply:
Hi @kurtmckee, thanks for your fast answer, It is the first time I'm making a PR, so wasn't sure what I should do. I modified the functions because I'm using views and materialized views in an app with flask_sqlalchemy and a postgres DB with several schemas. So I tested those (create_view, create_materialized_view and refresh with and without schema) in my app and they work properly. I can try to modify and add testing functions hopping that does not take much time.

Reply:
Great, thank you! Adding tests that demonstrate a bug is fixed -- or demonstrate a feature is working -- is an important part of development. It's particularly important for sqlalchemy-utils because the code owner, kvesteri, requires tests for all code-related PR's.

This particular function, `create_table_from_selectable()`, will cause the crash so please be sure that you're testing this function.

Reply:
Thanks, I already fix that bug, and the function create_table_from_selectable() is used by create_view() and create_materialized_view() so it is already tested

Reply:
The current PR contains a syntax error and will always crash. Perhaps you fixed it in a different branch, or fixed it after pushing the commits to the current branch?

Please add unit tests and open a new PR after further testing. Thanks!